src/ucp/core/ucp_ep.c

```diff
@@ -2487,7 +2487,8 @@ uct_ep_h ucp_ep_get_cm_uct_ep(ucp_ep_h ep)
 
 int ucp_ep_is_cm_local_connected(ucp_ep_h ep)
 {
-    return ucp_ep_has_cm_lane(ep) && (ep->flags & UCP_EP_FLAG_LOCAL_CONNECTED);
+    return ucp_ep_has_cm_lane(ep) && (ep->flags & UCP_EP_FLAG_LOCAL_CONNECTED) &&
```

Comment:
maybe we can remove ucp_ep_has_cm_lane(ep), rely only on (ucp_ep_get_cm_uct_ep(ep) != NULL)
@evgeny-leksikov WDYT?

Reply:
sure, `ucp_ep_get_cm_uct_ep` already contains `ucp_ep_get_cm_lane() == UCP_NULL_LANE` check